include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -99,6 +99,36 @@ ORT_DEFINE_RELEASE(ModelMetadata);
 ORT_DEFINE_RELEASE(ThreadingOptions);
 ORT_DEFINE_RELEASE(IoBinding);
 
+/// Ort::Float16_t/BFloat16_t are a structures that represents float16/bfloat16 data.
```

Comment:
nit: Does this conform to doxygen style?

Reply:
No, it does not. It conforms to XML, which was automatically suggested. I will make it Doxygen

---
In reply to: [518443567](https://github.com/microsoft/onnxruntime/pull/5716#discussion_r518443567) [](ancestors = 518443567)